recipes-bsp/u-boot/files/m100pfsevp/0001-riscv-dts-m100pfsevp-fix-broken-emmc-support.patch

```diff
@@ -0,0 +1,60 @@
+From f45add95a550741bed7aad5a1df62c78163cb079 Mon Sep 17 00:00:00 2001
+From: Wolfgang Grandegger <wg@aries-embedded.de>
+Date: Tue, 16 May 2023 16:56:08 +0200
+Subject: [PATCH 1/2] riscv: dts: m100pfsevp: fix broken emmc support
+
+To permit switching between SD card and eMMC, the clock freqency is
+limited to 50 Mhz. HS200 mode will make the eMMC inaccessible.
+
+Signed-off-by: Wolfgang Grandegger <wg@aries-embedded.de>
+---
+ arch/riscv/dts/mpfs-m100pfsevp.dts | 12 ++++--------
+ configs/m100pfsevp_defconfig       |  2 ++
+ 2 files changed, 6 insertions(+), 8 deletions(-)
+
+diff --git a/arch/riscv/dts/mpfs-m100pfsevp.dts b/arch/riscv/dts/mpfs-m100pfsevp.dts
+index cfca361eb6..84dbad50ea 100644
+--- a/arch/riscv/dts/mpfs-m100pfsevp.dts
++++ b/arch/riscv/dts/mpfs-m100pfsevp.dts
+@@ -48,17 +48,13 @@
+ 
+ &mmc {
+ 	status = "okay";
+-	bus-width = <4>;
++	bus-width = <8>;
++	max-frequency = <50000000>;
```

Comment:
Hi @grandwolf ,

Is there a reason why the max-frequency property is set to 50000000. The max frequency property should match the "MSS eMMC/SD/SDIO clock frequency" property in the MSS Configurator/XML which is fixed to a value of 200MHz.

Reply:
Hi @grandwolf , I'm bumping this message just in case it was missed. Would it be possible to get the latest MSS configurator XML or confirm that the "MSS eMMC/SD/SDIO clock frequency" value you are using in the XML is 50MHz?

Thanks

Reply:
Hello @vfalanis, I missed that message end of October, sorry!
Yes, the 50 MHz is mandatory. It will not work with 200 MHz.
The up-to-date XML file can be found here:

https://github.com/polarfire-soc/hart-software-services/blob/master/boards/aries-m100pfsevp/soc_fpga_design/xml/m100pfs_reference.xml

Hope that's what you are looking for !?

Reply:
@vfalanis, the reason for the 50 MHz is, that we are using a GPIO to switch between
SD card and eMMC.